website/src/types/model.ts

```diff
@@ -24,7 +24,7 @@ export type OutlineItem = {
 };
 
 /**
- * 本文情報
+ * Body content information.
```

Comment:
```suggestion
 * Details about the body of a documentation page.
```

(copied from the original `docs/src/model.rs`)